internal/handlers/tigris/msg_explain.go

```diff
@@ -28,11 +29,6 @@ import (
 
 // MsgExplain implements HandlerInterface.
 func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error) {
-	dbPool, err := h.DBPool(ctx)
```

Comment:
Oops I've encountered this while working on the same simplification in my PR. Although, I left DBPool creation just for healthcheck. Don't we want to tell user that connection with tigris can't be established when they run any command?

https://github.com/FerretDB/FerretDB/pull/1963/files#diff-59f3426f1c5e6f11f92ffbcad4c1d0ec18485d18f4bfab15e03652d711f24e52R32-R33

Reply:
@noisersup yes, sounds reasonable! I like your approach more. I'll close this PR.